packages/contracts-periphery/package.json

```diff
@@ -19,7 +19,8 @@
     "test": "yarn test:contracts",
     "test:contracts": "hardhat test --show-stack-traces",
     "test:forge": "forge test",
-    "test:coverage": "NODE_OPTIONS=--max_old_space_size=8192 hardhat coverage && istanbul check-coverage --statements 90 --branches 82 --functions 88 --lines 90",
+    "test:forge:coverage": "forge coverage",
+    "test:coverage": "NODE_OPTIONS=--max_old_space_size=8192 hardhat coverage",
```

Comment:
removed coverage thresholds because I couldn't (quickly) figure out how to combine hardhat and foundry coverage into a single coverage report

Reply:
Looks like this can be fixed using https://www.npmjs.com/package/istanbul-merge but seems out of scope for this pr